site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -0,0 +1,182 @@
+---
+layout: doc
+title: Link APIs
+description: The APIs for creating, retrieving, updating and deleting Identity Provider Links
+---
+
+== Overview
+
+[NOTE.since]
+====
+This API has been available since 1.28.0
+====
+
+This page contains the APIs that are used to manage Links that establish a relationship between a FusionAuth User and an Identity Provider.
+
+* <<Link a User>>
+* <<Complete a pending Link>>
+* <<Retrieve a Link>>
+* <<Unlink a User>>
+
+== Link a User
+
+This API is used to create a Link.
```

Comment:
This API is used to create a link between a FusionAuth user and a user in a 3rd party identity provider. This API may be useful when you already know the unique Id of a user in a 3rd party identity provider and the corresponding FusionAuth user.